tooling/nargo_fmt/src/chunks.rs

```diff
@@ -803,7 +809,7 @@ impl<'a> Formatter<'a> {
                 // )
                 let comma_trimmed = self.trim_comma();
                 self.decrease_indentation();
-                self.write_line();
+                self.write_line_without_skipping_whitespace_and_comments();
```

Comment:
When do we want to write a line that skips comments?

Reply:
Oh, by default `write_line` will skip any comments and whitespace that exist at the current lexer position. However, here we are writing a bunch of chunks/groups which means that we already lexed the content we want to write. If we call`write_line` here we might be skipping whitespace/comment that we don't want to skip just yet. So this is a bug fix (though I didn't find code that breaks with `write_line`, but I thought it would be best to use the correct method).